### DISCUSSION TITLE
Disambiguate "decentralized identifier"

### DISCUSSION THREAD

**Discussion by Oskar-van-Deventer** [trustoverip/trust-spanning-protocol#11] [General]
When reading [Trust over IP (ToIP) Technology Architecture Specification]( https://github.com/trustoverip/TechArch/blob/main/spec.md), the concept "decentralized identifier" may cause confusion. At some places, it is described as a generic concept with some properties/criteria that distinguishes it from identifiers that are not "decentralized" identifiers. At some places, the term is used synonymously with "W3C DIDs". Some examples of it are definitely not W3C DID. And some are not, but can be encapsulated as a W3C DID. Moreover, the term "decentralised identifier" is used next to "DID", where it is unclear whether they are synonymous (DID being the abbreviation) or not.

As the new ToIP TSP spec will be based on that architecture spec, and the TSP spec will need to be clear and implementable to outsiders, it may be useful to alacritously disambiguate this terminology confusion, e.g. with the help of the ToIP Concepts and Terminology working group.

Oskar

**Comment by talltree** [comment]
@Oskar-van-Deventer Excellent point. As a co-editor of the W3C Decentralized Identifiers (DIDs) 1.0 spec, I tried to be consistent about the use of the terms "decentralized identifier" and "DID" in the [ToIP Technology Architecture Specification V1.0](https://github.com/trustoverip/TechArch/blob/main/spec.md) (TAS), but clearly that needs an editing scrub.

This should be raised as [an issue on the TAS](https://github.com/trustoverip/TechArch/issues). Do you want to raise that issue, or should I?
